src/components/search/search.njk

```diff
@@ -14,22 +14,26 @@
   <h2>Big</h2>
 
   <form class="usa-search usa-search--big" role="search">
-    <label class="usa-sr-only" for="search-field-big">Search</label>
-    <input class="usa-input" id="search-field-big" type="search" name="search">
-    <button class="usa-button" type="submit">
-      <span class="usa-search__submit-text">Search</span>
-    </button>
+    <div>
```

Comment:
Looks like this in fractal for me:
![image](https://user-images.githubusercontent.com/3385219/76097809-f396df80-5f8d-11ea-97c0-11382cd6f78f.png)

We need to add the role to the `div`, remove the div, or update the styling.